api/lds.proto

```diff
@@ -87,7 +87,14 @@ message FilterChainMatch {
 // and related parameters.
 message FilterChain {
   FilterChainMatch filter_chain_match = 1;
-  DownstreamTlsContext tls_context = 2;
+
+  oneof transport_security {
+    DownstreamTlsContext tls_context = 2;
```

Comment:
While you're here, can you add comment here, similar to the one for `UpstreamTlsContext`?

Reply:
Done